app/controllers/application_controller.rb

```diff
@@ -1,4 +1,12 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  rescue_from StandardError, with: :render_internal_server_error
+
+  private
+
+  def render_internal_server_error
+    render file: Rails.root.join('public/500.html'), status: :internal_server_error,
+           layout: false, content_type: 'text/html'
+  end
```

Comment:
開発中にエラーみたかったらデバッガを使うかコメントアウトするしかないですか？(少し不便そうだと思った)

Reply:
あー、そういうことは考えなかったですね。

Reply:
ちなみに @Taku3939 だったらどうすると思いますか。

Reply:
僕は`if Rails.env == 'production'`とかで開発環境ではログが表示されるようにするくらいしか思いつかないですね

Reply:
> Using rescue_from with Exception or StandardError would cause serious side-effects as it prevents Rails from handling exceptions properly. As such, it is not recommended to do so unless there is a strong reason.

https://guides.rubyonrails.org/action_controller_overview.html#rescue-from

なるほど。安易にStandardErrorをrescue_fromで補足するべきではないようだ。実際、 @Taku3939 に指摘されたような影響があるわけで。

Reply:
rescue_fromによるStandardErrorの補足を削除して、アクション内で必要な例外のみを捕捉するように変更しました。